clap_complete/tests/testsuite/engine.rs

```diff
@@ -682,14 +682,23 @@ fn suggest_multi_positional() {
     let mut cmd = Command::new("dynamic")
         .arg(
             clap::Arg::new("positional")
-                .value_parser(["pos_1, pos_2, pos_3"])
-                .index(1),
+                .value_parser(["pos_1"])
+                .index(1)
+                .required(true),
         )
         .arg(
             clap::Arg::new("positional-2")
                 .value_parser(["pos_a", "pos_b", "pos_c"])
                 .index(2)
-                .num_args(3),
+                .num_args(3)
+                .required(true),
+        )
+        .arg(
+            clap::Arg::new("positional-3")
+                .value_parser(["pos_d", "pos_e"])
```

Comment:
The values were chosen to make it obvious which positional argument they are associated with.  If you are reusing this test, we should probably change things to make that clearer, like adding a refactor commit before this one that changes the values to `pos_<num>_<letter>` so `positional` has `["pos_1_a"]`, `positional-2` has `["pos_2_a", "pos_2_b"]`.

Then in this commit, you'd have `["pos_3_a", "pos_3_b"]`.